### PR TITLE
Adjust system role for KVM

### PR DESCRIPTION
**Pull request by aginies**
- enable kdump/firewalld (jsc#SLE-17265)
- add requires on enhanced_base (bsc#1179177)
- 15.1.3

### CODE REVIEW

File: package/system-role-kvm.spec [hunk 1]
```diff
@@ -36,7 +36,7 @@ BuildRequires:  yast2-installation-control >= 4.0.0
 
 Url:            https://github.com/yast/system-role-kvm
 AutoReqProv:    off
-Version:        15.1.2
+Version:        15.1.3
```
Comment (jreidinger) [inline package/system-role-kvm.spec:39]: Hi, yast/products control/system roles does not follow symver but product based version. So if target of this change is SLE 15 SP3 then proper version is 15.3.0 ( so SLE15 SP3 0 as first version in given code stream ).

Comment (jreidinger) [reply package/system-role-kvm.spec:39]: But beside this, rest of change looks good.

File: package/system-role-kvm.changes [hunk 2]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Jan  5 17:00:08 CET 2021 - aginies <aginies@suse.com>
+- enable kdump/firewalld (jsc#SLE-17265)
+- add requires on enhanced_base (bsc#1179177)
+- 15.1.3
```
Comment (jreidinger) [inline package/system-role-kvm.changes:5]: and here

Comment (aginies) [reply package/system-role-kvm.changes]: done

**Review by jreidinger — APPROVED** [review]

**Pull request merged**